src/openforms/tests/e2e/input_validation_base.py

```diff
@@ -133,7 +137,7 @@ def _assertBackendValidation(self, form: Form, key: str, value: JSONValue):
                 "step_uuid": step.uuid,
             },
         )
-        body = {"data": {key: value}}
+        body = {"data": {key: value}} if value is not None else {"data": {}}
```

Comment:
this will cause issues because for number fields `null` (and thus `None` in Python) is valid and should not be omitted.